add new CLI commands to webapp tutorial

emily-potyraj opened the pull request:
- adds "Inspect application" section to Webapp tutorial (but not Dapr microservices tutorial)
  - rad deployment list --application-name webapp
  - rad component get --application-name webapp --name todoapp
- updates log messages in `rad deploy` to match capitalized log messages in `rad env init`
- minor flow change to microservices tutorial: move "tour completed" text to the end

Commit:
add new CLI commands to webapp tutorial

AaronCrawfis reviewed (changes requested):
Couple small changes on tutorial steps to generalize subscription ID and resource group.

Also made some suggestions on emojis in the logger. We've gotten positive feedback from Dapr on this. Open to discussion here.

Commit:
- add the new CLI commands to dapr microservices tutorial as well
- intersperse "check your deployment" type commands within the other steps (instead of previous
  separate "inspect the app" section)
- update entire tutorial text for 1) reiforcing concepts language and 2) decreasing verbosity

emily-potyraj committed:
Update cmd/cli/cmd/deploy.go

Co-authored-by: Aaron Crawfis <Aaron.Crawfis@microsoft.com>

Commit:
Add Aaron's suggested changes & remove explicit statestore test from webapp tutorial.
